multi-node/emulate-multi-node.md

```diff
@@ -7,191 +7,200 @@ We use the `deepspeed` launcher here. There is no need to actually use any of th
 The full setup instructions follow:
 
 1. Create a `hostfile`:
-```
-$ cat hostfile
-worker-0 slots=1
-worker-1 slots=1
-```
+   
+    ```bash
```

Comment:
@stas00 do you mean e.g. the indentation (4 spaces) before the lines wrapped in backticks?


```markdown
1. Create a `hostfile`:

    ```bash
    $ cat hostfile
    [...]
    ```
```

Reply:
that's what I mean - yes - you wanted to do `s/```/```bash/` but ended up making massive formatting changes to the doc.

Reply:
ahh yes okay. 

I tried keeping consistent indentation between the numbered list items and their contents so that 

---

1. First list item

    ```bash
    $ cat hostfile
    ```
    
    blah blah blah

vs.

---

1. First list item

```bash
$ cat hostfile
```

blah blah blah

---

but I can definitely remove the indentation throughout, no problem

Reply:
Removed in [d1476ec](https://github.com/stas00/ml-engineering/pull/17/commits/d1476ecbec941ffe12bd7ca18ee22645d8e23e19)

Reply:
thank you for explaining the purpose of that particular white space, Sam. 

I read those documents in raw .md most of the time I don't see the difference.

I'm not sure what to do here. It does make the final rendering better, but leads to more complicated maintenance with needing to manage leading space consistently.

Let's perhaps leave it as is for now and later when things are more complete - do this items change consistently across all of the files.

It's too bad there no other way to signal to md renderer when a section starts and ends and not via leading white-space. I have used it before and it's a hell of maintenance - especially if there are nested sections.

Reply:
yeah for the sake of consistency with the rest of the text I'm happy to leave it as is for now

but yeah obviously its your repo / content so I'm on board with whatever you think, formatting-wise 🤷🏻‍♂️